test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -190,126 +190,121 @@ def test_make_xsd_id_compatible(self) -> None:
 
     def test_find_date_in_string_iso(self) -> None:
         """template: 2021-01-01"""
-        self.assertEqual(excel2xml.find_date_in_string("x 1492-10-12, x"), "GREGORIAN:CE:1492-10-12:CE:1492-10-12")
-        self.assertEqual(excel2xml.find_date_in_string("x 0476-09-04. x"), "GREGORIAN:CE:0476-09-04:CE:0476-09-04")
-        self.assertEqual(excel2xml.find_date_in_string("x (0476-09-04) x"), "GREGORIAN:CE:0476-09-04:CE:0476-09-04")
-        self.assertEqual(excel2xml.find_date_in_string("x [1492-10-32?] x"), None)
+        assert excel2xml.find_date_in_string("x 1492-10-12, x") == "GREGORIAN:CE:1492-10-12:CE:1492-10-12"
+        assert excel2xml.find_date_in_string("x 0476-09-04. x") == "GREGORIAN:CE:0476-09-04:CE:0476-09-04"
+        assert excel2xml.find_date_in_string("x (0476-09-04) x") == "GREGORIAN:CE:0476-09-04:CE:0476-09-04"
+        assert excel2xml.find_date_in_string("x [1492-10-32?] x") is None
 
     def test_find_date_in_string_eur_date(self) -> None:
         """template: 31.4.2021 | 5/11/2021 | 2015_01_02"""
-        self.assertEqual(excel2xml.find_date_in_string("x (30.4.2021) x"), "GREGORIAN:CE:2021-04-30:CE:2021-04-30")
-        self.assertEqual(excel2xml.find_date_in_string("x (5/11/2021) x"), "GREGORIAN:CE:2021-11-05:CE:2021-11-05")
-        self.assertEqual(excel2xml.find_date_in_string("x ...2193_01_26... x"), "GREGORIAN:CE:2193-01-26:CE:2193-01-26")
-        self.assertEqual(excel2xml.find_date_in_string("x -2193_01_26- x"), "GREGORIAN:CE:2193-01-26:CE:2193-01-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 2193_02_30 x"), None)
+        assert excel2xml.find_date_in_string("x (30.4.2021) x") == "GREGORIAN:CE:2021-04-30:CE:2021-04-30"
+        assert excel2xml.find_date_in_string("x (5/11/2021) x") == "GREGORIAN:CE:2021-11-05:CE:2021-11-05"
+        assert excel2xml.find_date_in_string("x ...2193_01_26... x") == "GREGORIAN:CE:2193-01-26:CE:2193-01-26"
+        assert excel2xml.find_date_in_string("x -2193_01_26- x") == "GREGORIAN:CE:2193-01-26:CE:2193-01-26"
+        assert excel2xml.find_date_in_string("x 2193_02_30 x") is None
 
     def test_find_date_in_string_eur_date_range(self) -> None:
         """template: 27.-28.1.1900"""
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-26.2.0800 x"), "GREGORIAN:CE:0800-02-25:CE:0800-02-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 25. - 26.2.0800 x"), "GREGORIAN:CE:0800-02-25:CE:0800-02-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-24.2.0800 x"), None)
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-25.2.0800 x"), None)
+        assert excel2xml.find_date_in_string("x 25.-26.2.0800 x") == "GREGORIAN:CE:0800-02-25:CE:0800-02-26"
+        assert excel2xml.find_date_in_string("x 25. - 26.2.0800 x") == "GREGORIAN:CE:0800-02-25:CE:0800-02-26"
+        assert excel2xml.find_date_in_string("x 25.-24.2.0800 x") is None
+        assert excel2xml.find_date_in_string("x 25.-25.2.0800 x") is None
```

Comment:
I don't see why this is not a valid entry. It is a time interval of one day, which is something we also do.